src/backend/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+   extends: '@senecacdot/eslint-config-telescope',
+   overrides: [
+     {
+       files: ['src/backend/**/*.js', 'tools/**/*.js', 'src/api/**/*.js'],
```

Comment:
If we're already in the `src/backend` directory, we don't need `tools/*` and `src/api/*`